app/models/collection_item.rb

```diff
@@ -187,10 +187,9 @@ def send_work_invitation
 
   after_update :notify_of_status_change
   def notify_of_status_change
-    if saved_change_to_unrevealed?
-      # making sure notify_recipients in the work model has not already notified 
-      # the user
-      if !work.new_recipients.blank?
+    if saved_change_to_unrevealed? && item.respond_to?(:new_recipients)
+      # making sure that creation_observer.rb has not already notified the user
```

Comment:
I think this comment might be a little outdated -- creation_observer no longer exists.